tealeaf-service/tealeaf-api/build.gradle

```diff
@@ -1,3 +1,3 @@
 dependencies {
-    implementation 'jakarta.validation:jakarta.validation-api:3.0.2'
+    implementation 'jakarta.validation:jakarta.validation-api:3.1.0'
```

Comment:
Let me deal with this separately after upgrading the dependencies without this change.